test/onnx/test_fx_passes.py

```diff
@@ -154,6 +155,22 @@ def forward(self, x, y):
         self.assertIn("torch_nn_modules_activation_ReLU_relu_1", function_proto_names)
         self.assertIn("torch_nn_modules_activation_ReLU_relu_2", function_proto_names)
 
+        # exported program
```

Comment:
non blocking: the test could be parameterized. This way, even with modularization fails for the nn.Module, it would still tests the ExportedProgram test